src/controllers/MLFacetQueryController.ts

```diff
@@ -47,6 +55,9 @@ export class MLFacetQueryController {
     };
 
     queryBuilder.facetRequests.push(facetRequest);
+    if (this.freezeFacetOrder) {
```

Comment:
Is the condition needed or would adding `queryBuilder.freezeFacetOrder = false;` be okay?

Reply:
No because this code will run multiple facets